build/configs/esp32_DevKitC/kernel_tc/defconfig

```diff
@@ -0,0 +1,839 @@
+# CONFIG_LIBC_STRERROR is not set
+# CONFIG_LIBC_PERROR_STDOUT is not set
+CONFIG_LIBC_TMPDIR="/tmp"
+CONFIG_LIBC_MAX_TMPFILE=32
+CONFIG_ARCH_LOWPUTC=y
+# CONFIG_LIBC_LOCALTIME is not set
+# CONFIG_TIME_EXTENDED is not set
+CONFIG_LIB_SENDFILE_BUFSIZE=512
+# CONFIG_ARCH_OPTIMIZED_FUNCTIONS is not set
+
+#
+# Non-standard Library Support
+#
+
+#
+# Basic CXX Support
+#
+# CONFIG_C99_BOOL8 is not set
+# CONFIG_HAVE_CXX is not set
+
+#
+# External Libraries
+#
+# CONFIG_AVS_DEVICE_SDK is not set
+# CONFIG_AWS_SDK is not set
+# CONFIG_NETUTILS_CODECS is not set
+
+#
+# CURL Options
+#
+# CONFIG_ENABLE_CURL is not set
+# CONFIG_ERROR_REPORT is not set
+# CONFIG_ENABLE_IOTIVITY is not set
+# CONFIG_NETUTILS_JSON is not set
+# CONFIG_LIBTUV is not set
+# CONFIG_STRESS_TOOL is not set
+# CONFIG_VOICE_SOFTWARE_EPD is not set
+
+#
+# Application Configuration
+#
+
+#
+# Application entry point list
+#
+# CONFIG_ENTRY_MANUAL is not set
+CONFIG_ENTRY_HELLO=y
+# CONFIG_ENTRY_TESTCASE is not set
+CONFIG_USER_ENTRYPOINT="hello_main"
+CONFIG_BUILTIN_APPS=y
+
+#
+# Examples
+#
+# CONFIG_EXAMPLES_AVS_TEST is not set
+# CONFIG_EXAMPLES_AWS is not set
+# CONFIG_EXAMPLES_CURLTEST is not set
+# CONFIG_EXAMPLES_EEPROM_TEST is not set
+# CONFIG_EXAMPLES_EVENTLOOP is not set
+# CONFIG_EXAMPLES_FOTA_SAMPLE is not set
+# CONFIG_FILESYSTEM_HELPER_ENABLE is not set
+CONFIG_EXAMPLES_HELLO=y
+# CONFIG_EXAMPLES_HELLOXX is not set
+# CONFIG_EXAMPLES_IOTBUS_TEST is not set
+# CONFIG_EXAMPLES_IOTJS_STARTUP is not set
+# CONFIG_EXAMPLES_KERNEL_SAMPLE is not set
+# CONFIG_EXAMPLES_LIBTUV is not set
+# CONFIG_EXAMPLES_NETTEST is not set
+# CONFIG_EXAMPLES_PROC_TEST is not set
+# CONFIG_EXAMPLES_SELECT_TEST is not set
+# CONFIG_EXAMPLES_SENSORBOARD is not set
+# CONFIG_EXAMPLES_SETJMP_TEST is not set
+# CONFIG_EXAMPLES_SMART is not set
+# CONFIG_EXAMPLES_SMART_TEST is not set
+# CONFIG_EXAMPLES_SPEECH_DETECTOR_TEST is not set
+# CONFIG_EXAMPLES_ST_THINGS is not set
+CONFIG_EXAMPLES_TESTCASE=y
+# CONFIG_DISABLE_MANUAL_TESTCASE is not set
+# CONFIG_EXAMPLES_TESTCASE_DRIVERS is not set
+# CONFIG_EXAMPLES_TESTCASE_FILESYSTEM is not set
+CONFIG_EXAMPLES_TESTCASE_KERNEL=y
+# CONFIG_DEBUG_TC_KN is not set
+# CONFIG_TC_KN_ALL is not set
+CONFIG_TC_KERNEL_CLOCK=y
+CONFIG_TC_KERNEL_ENVIRON=y
+CONFIG_TC_KERNEL_ERRNO=y
+# CONFIG_TC_KERNEL_GROUP is not set
+CONFIG_TC_KERNEL_LIBC_FIXEDMATH=y
+CONFIG_TC_KERNEL_LIBC_INTTYPES=y
+CONFIG_TC_KERNEL_LIBC_LIBGEN=y
+CONFIG_TC_KERNEL_LIBC_MATH=y
+CONFIG_TC_KERNEL_LIBC_MISC=y
+CONFIG_TC_KERNEL_LIBC_MQUEUE=y
+CONFIG_TC_KERNEL_LIBC_PTHREAD=y
+CONFIG_TC_KERNEL_LIBC_QUEUE=y
+CONFIG_TC_KERNEL_LIBC_SCHED=y
+CONFIG_TC_KERNEL_LIBC_SEMAPHORE=y
+CONFIG_TC_KERNEL_LIBC_SIGNAL=y
+CONFIG_TC_KERNEL_LIBC_STDIO=y
+CONFIG_TC_KERNEL_LIBC_STDLIB=y
+CONFIG_TC_KERNEL_LIBC_STRING=y
+CONFIG_TC_KERNEL_LIBC_TIMER=y
+CONFIG_TC_KERNEL_LIBC_UNISTD=y
+CONFIG_TC_KERNEL_LIBC_SYSLOG=y
+CONFIG_TC_KERNEL_MQUEUE=y
+CONFIG_TC_KERNEL_PTHREAD=y
+# CONFIG_TC_KERNEL_ROUNDROBIN is not set
```

Comment:
Some TCs are disabled, such as roundrobin ,tash heapinfo and so on.

Reply:
@jeongchanKim Thanks for reminding us.

    Current defconfig file is created by "make menuconfig" operation. The HEAPINFO, STACKMONITOR, MEMORY_SAFETY, TERMIOS are closed in the default .config file which is created through menuconfig.
    Now, we enable the HEAPINFO, STACKMONITOR, MEMORY_SAFETY, but keep ROUND ROBIN 
 & TERMIOS  closed.  Becasue TERMIOS would be failed due to lack of some symbols in linking. And ROUND ROBIN is closed for we meeting crash issue in pthread stack overflow.
    We will check and fixed crash issue in round robin and the failed TC. 

Thanks.